standard/operator.lua

```diff
@@ -67,11 +67,14 @@ end
 ---@param item string|number
 ---@return fun(tbl: table): any
 function Operator.property(item)
-	if string.find(item, '%.') then
-		error('Pathing not yet supported in property')
-	end
+	local pathSegments = mw.text.split(item, '.', true)
+
 	return function(tbl)
-		return tbl[item]
+		local selected = tbl
+		for _, pathSegment in ipairs(pathSegments) do
+			selected = (selected or {})[pathSegment]
```

Comment:
What's the general sentiment on letting this error in case a table does not exist?
i.e.
```suggestion
			selected = selected[pathSegment]
```

Reply:
i see 3 options:
- do not let it error, just return `nil` (as suggested in this PR)
- add `assert(tbl ~= nil, ....)`, but do not let it error if subtables on the path do not exist
- let it error on every missing (sub) table on the path (as per the code suggestion above)

personally i am indifferent to it

Reply:
I'll keep this in draft mode until this is decided and the test could be added.

Reply:
Paging @Rathoz for an opinion here

Reply:
The initial idea was based on Python's Operator library. 
In Python, it will do what @mbergen suggested.
I'm open for arguments

Reply:
Always erroring would be the only option that does not change the current behavior.
IMO this would be preferable although errorhandling in lua is different to python.
It might be a good idea to "catch" the segment where it happens to provide a better error output than the usual "Attempt to index nil".

Note that i'm a bit biased since most of my work stuff is in python currently.